projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -128,6 +137,12 @@ export default function SubscribePanels() {
 						withModal={ false }
 					/>
 				) }
+
+				{ ! isNewsletterFeatureEnabled() && (
+					<Button variant="link" href={ DISCUSSION_URL }>
+						{ __( 'Enable newsletter', 'jetpack' ) }
```

Comment:
@Automattic/gold Do you have any opinions on what we should display when Newsletters are disabled?

Here's the issue we're trying to solve: https://github.com/Automattic/jetpack/issues/28288

Reply:
>  Do you have any opinions on what we should display when Newsletters are disabled?

Probably some kind of informational text?

Reply:
"The newsletter functionality is currently disabled. Enable it to send your posts to your email followers as a newsletter" 
Or something like that

Reply:
CC @mtias